### PR TITLE
linter fixes

### PR DESCRIPTION
**Pull request by e111077**
Hooli linter fixes

### CODE REVIEW

**Comment by e111077** [comment]
tests never worked in the 1.x branch of app-storage.

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js [hunk 1]
```diff
@@ -241,7 +241,7 @@
         this.handleClientMessage(event, port)
       }.bind(this));
 
-      if (!port in this[CLIENT_PORTS]) {
+      if (!(port in this[CLIENT_PORTS])) {
```
Comment (rictic) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: woah, this looks like a real bug!

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js [hunk 2]
```diff
@@ -310,6 +310,6 @@
   self.appIndexedDBMirrorWorker = new AppIndexedDBMirrorWorker();
 
   self.addEventListener('connect', function(event) {
-    appIndexedDBMirrorWorker.registerClient(event.ports[0])
+    self.appIndexedDBMirrorWorker.registerClient(event.ports[0])
```
Comment (rictic) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: this too, I think?

**Comment by rictic** [comment]
travis is unhappy because this repo doesn't have a travis file?

**Comment by e111077** [comment]
Yeah, this repo is kinda just a transitionary repo until we merge 2.0-preview which has all the bells and whistles

**Comment by rictic** [comment]
rgr, LGTM

**Pull request merged**